webui/react/src/pages/ExperimentDetails/ExperimentCheckpoints.tsx

```diff
@@ -128,12 +126,13 @@ const ExperimentCheckpoints: React.FC<Props> = ({ experiment, pageRef }: Props)
     [registerModal],
   );
 
-  const handleDelete = useCallback((checkpoints: string[]) => {
-    readStream(
```

Comment:
qq: do you know why it used to use `readStream`

Reply:
the copy implies that we purposefully dropped the error messaging -- i think that's why we used readstream, because its default behavior is to drop errors